src/System.Net.Http/src/System/Net/Http/Headers/RangeHeaderValue.cs

```diff
@@ -82,9 +82,9 @@ public override string ToString()
                         sb.Append(", ");
                     }
 
-                    sb.Append(range.From);
+                    if (range.From.HasValue) sb.Append(range.From.GetValueOrDefault());
```

Comment:
nit: curly braces?
nit: you already check with HasValue, why GetValueOrDefault instead of Value?

Reply:
> you already check with HasValue, why GetValueOrDefault instead of Value?

Because I already checked HasValue.  Value has to do `if (!hasValue) throw ...; return value;` whereas GetValueOrDefault just does `return value;`.

Reply:
Heh, I did not know that, but makes sense.
Thanks.